cli/pack/templates/app-inst-unit-template.txt

```diff
@@ -5,6 +5,7 @@ After=network.target
 [Service]
 Type=forking
 ExecStart={{ .TT }} -L {{ .ConfigPath }} start %i
+ExecStop={{ .TT }} -L {{ .ConfigPath }} stop %i
```

Comment:
I think, we must use --cfg option instead of -L. Because -L changes current dir to the target directory.